### DISCUSSION TITLE
Some ALOS FNF assets are missing

### DISCUSSION THREAD

**Discussion by mikemahoney218** [microsoft/PlanetaryComputer#367] [Q&A]
Hi all! 

A user alerted me (https://github.com/Permian-Global-Research/rsi/issues/74) that my tool was failing to download some data from the ALOS FNF Mosaic product: https://planetarycomputer.microsoft.com/dataset/alos-fnf-mosaic

On investigation, it seems that some STAC items link to assets that don't exist. For instance, here's a valid item:
https://planetarycomputer.microsoft.com/api/stac/v1/collections/alos-fnf-mosaic/items/N37W082_16_FNF

But the C asset href leads to an error:
https://pceo.blob.core.windows.net/palsar/alos_fnf_mosaic/N37W082_16_C_F02DAR.tif

This appears to be impacting multiple items.

**Comment by mikemahoney218** [comment]
I'm noticing that in both my testing and in the initial report, the issues seem to be from 2015 and 2016 -- I'm wondering if those two years may be broadly impacted.

**Reply by TomAugspurger** [reply]
Thanks for the report. We'll take a closer look.
